src/Runner.php

```diff
@@ -652,6 +652,39 @@ public function processFile($file)
             }
         } catch (\Exception $e) {
             $error = 'An error occurred during processing; checking has been aborted. The error message was: '.$e->getMessage();
+
+            // Determine which sniff caused the error.
+            $sniffStack = null;
+            $nextStack  = null;
+            foreach ($e->getTrace() as $step) {
+                if (isset($step['file']) === false) {
+                    continue;
+                }
+
+                if (substr($step['file'], -9) === 'Sniff.php') {
+                    $sniffStack = $step;
+                    continue;
+                }
+
+                if (empty($sniffStack) === false) {
```

Comment:
why not?
```suggestion
                if (!empty($sniffStack)) {
```

or even more simply

```suggestion
                if ($sniffStack) {
```

both are more readable

Reply:
Oh I see, I was not aware of this sorry, it's a bit strange because it's less readable and less efficient.

Reply:
🤷🏻‍♀️